### DISCUSSION TITLE
Colorscapes Update

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#3403] [Ideas]
### Idea Description

Not a fan of the horizontal gradient that the update has caused to produce. It was better before.

![Screenshot from 2024-11-25 03-23-12](https://github.com/user-attachments/assets/d0a66b6e-db9e-420d-9e4c-41d6d556c241)

### Idea Category

API

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by maximo1984** [comment]
Rather than being subtle, it jumps out as distracting (not even considering the stark colors on this example, but all) Another example. A top down vertcial gradient would be less instrusive. Or a top left to bottom right gradient. Need more padding between colors.
![Screenshot from 2024-11-25 03-25-44](https://github.com/user-attachments/assets/4ef4371e-9b2a-4d0c-8d35-4fca342143b2)

**Comment by majora2007** [comment]
I have more to do, I found a much better rendering technique but it was too close to the release.

You can turn it off with a theme by setting the css variable to false.

**Comment by maximo1984** [comment]
I am still organizing my collection, tagging and importing so its not a big deal right now. I just want to point it out since this was just released. Thanks. Also thanks for implementing some of my prior discussion ideas. Proud to be a contributor towards this project.
